docs/project/contribution_tools.md

```diff
@@ -222,6 +222,9 @@ get `$PATH` changes.
 It may be necessary to run `bazel clean` after updating versions in order to
 clean up cached state.
 
+macOS only: In case of "llvm-ar no such file or directory" error see
+[llvm-fix](https://tipseason.com/carbon-language-execvp-error/)
```

Comment:
On line 219, we already have a macos only note. Since the linked tip seems to match the macos install, maybe it'd be better to add there something like "System installs of macOS typically won't work, for example being an old LLVM version or missing llvm-ar; [setup commands](#setup-commands) includes LLVM from Homebrew for this reason."

Reply:
Ok, you think that I should integrate the bash file code or leave the link to the guide?